lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
+  var lib = './../';
```

Comment:
The separate "./../" variable is also kind of bothering me, but that's entirely a personal coding preference thing on my end.